### ISSUE TITLE
Deleted version v1.5.5 causes problems should be retracted

### ISSUE THREAD

**Issue by J-Rocke** [go-gorm/postgres#247]
## GORM Playground Link

Not applicable to this kind of issue. Just go to the main branch and run `go get -u gorm.io/driver/postgres`

## Description

This is a follow up to my comment here: https://github.com/go-gorm/postgres/issues/244#issuecomment-1926716459

v1.5.5 was released, had bug and was deleted. This did not follow the procedure how such things should be handled which can be found here https://go.dev/ref/mod#go-mod-file-retract

This lead to the go tooling and https://pkg.go.dev/gorm.io/driver/postgres to still list v1.5.5 as newest version.
If i run `go get -u ./...` in a module using `gorm.io/driver/postgres` it still pulls v.1.5.5 since the go module mirror still servers it even after it beeing deleted.

## Proposed fix.

Release a new version properly retracting the v1.5.5. How to can be found here: https://go.dev/ref/mod#go-mod-file-retract

**Comment by garrettladley** [comment]
+1 on this @J-Rocke , one of my repositories has dependabot and it prompted us to bump to 1.5.5 ([see](https://github.com/GenerateNU/sac/pull/157))

**Comment by iTanken** [comment]
https://go.dev/ref/mod#go-mod-file-exclude

```go.mod
require (
	gorm.io/driver/postgres v1.5.4
)

exclude gorm.io/driver/postgres v1.5.5 // This release is a breaking change
```

**Comment by J-Rocke** [comment]
Fixed by v1.5.6 Thanks at @jinzhu  and @black-06 for the quick resolution.
